openmls/src/group/core_group/staged_commit.rs

```diff
@@ -282,6 +282,7 @@ impl CoreGroup {
     ///
     /// This function should not fail and only returns a [`Result`], because it
     /// might throw a `LibraryError`.
+    #[cfg(any(feature = "test-utils", test))]
     pub fn merge_commit(&mut self, staged_commit: StagedCommit) -> Result<(), CoreGroupError> {
```

Comment:
I'm a bit confused as to why we can hide this function behind a cfg. Shouldn't we be using this function in `MlsGroup` somewhere to merge staged commits?

Reply:
This is another one of these issues. (See my general comment.)

Reply:
Again, #645.